roles/edpm_reboot/tasks/main.yaml

```diff
@@ -65,16 +70,17 @@
 
 - name: Print message if reboot is required, but is not going to be started
   ansible.builtin.debug:
-    msg: "Reboot is required but was not started. User has to plan the reboot and set edpm_reboot_force_reboot to true"
+    msg: "Reboot is required but was not started. Edpm_reboot_strategy is set to never or this is
+          already deployed machine. Reboot has to be planned. To start reboot set edpm_reboot_strategy
+          to force"
   when:
     - reboot_required|default(false)
-    - nova_conf_check.stat.exists
-    - not edpm_reboot_force_reboot|bool
+    - (edpm_reboot_strategy|lower == "never" or old_tripleo_node_conf.stat.exists or edpm_node_conf.stat.exists)
 
 - name: Run reboot nodes actions
-  when:
-    - reboot_required is defined and reboot_required|bool
-    - edpm_reboot_force_reboot|bool or not nova_conf_check.stat.exists
+  when: (edpm_reboot_strategy|lower == "force") or
+        (reboot_required|default(false) and edpm_reboot_strategy|lower == "auto" and not old_tripleo_node_conf.stat.exists
```

Comment:
does it need to check not just the old but the new nova conf existence?

Reply:
It does check in next line.83 (and not edpm_node_conf.stat.exists)
Strictly speaking it checks for any openstack conf that may exists in config paths, because there was an ask to make check more node type agnostic.

Reply:
No problem, thank you for reviewing change.